lib/aws/AwsSigv4Signer.js

```diff
@@ -77,6 +77,12 @@ function AwsSigv4Signer(opts = {}) {
     request.region = opts.region;
     request.headers = request.headers || {};
     request.headers['host'] = request.hostname;
+    if (opts.service === 'aoss') {
```

Comment:
I am strongly opposed to this. The server is aligning the implementation.

Reply:
Adding x-ams-context-sha256 works for both :)

Reply:
Amazing!!